### PR TITLE
Update Spanish Translation

### PR DESCRIPTION
**Pull request by tomaja18**
Update Spanish Translation

### CODE REVIEW

**Comment by EleoXDA** [comment]
Please remove "untranslatable" lines from translation. We have to remove them everytime manually.

File: app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 <resources>
 	<!--General strings-->
+	<string name="app_name" translatable="false">AOSP Mods</string>
```
Comment (EleoXDA) [inline app/src/main/res/values-es/strings.xml]: Please remove all the "untranslatable" lines from translation. We have to remove them everytime manually.

**Review by EleoXDA — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-es/strings.xml (hunk 1 above)
Comment (tomaja18) [reply app/src/main/res/values-es/strings.xml]: It is already corrected

**Comment by EleoXDA** [comment]
Why did you close the pull request? Please reopen it so that we can merge it

**Comment by tomaja18** [comment]
I think that's it, I don't know the platform very well, sorry.

**Comment by tomaja18** [comment]
Ok, I think that is now correct.

**Review by EleoXDA — APPROVED** [review]
Ok

**Pull request merged**